core/dbt/parser/schemas.py

```diff
@@ -544,15 +547,22 @@ def parse_file(self, block: FileBlock, dct: Dict = None) -> None:
 
 def check_format_version(file_path, yaml_dct) -> None:
     if "version" not in yaml_dct:
-        raise_invalid_schema_yml_version(file_path, "no version is specified")
+        raise_invalid_property_yml_version(
+            file_path, "the yml property file {} is missing a version tag".format(file_path)
+        )
 
     version = yaml_dct["version"]
     # if it's not an integer, the version is malformed, or not
     # set. Either way, only 'version: 2' is supported.
     if not isinstance(version, int):
-        raise_invalid_schema_yml_version(file_path, "the version is not an integer")
+        raise_invalid_property_yml_version(
+            file_path,
+            "its 'version:' tag must be an integer value. {} is not an integer".format(version),
```

Comment:
Minor wording suggestion: give examples of the full line we expect like `version: 2`

Reply:
Alright, so I thought about this myself hahaha. At what point should the code just say "if it isn't equal to two, fail it." I can see where it's useful though from a word standpoint now.